specsVMware/portworx-local-install.yaml

```diff
@@ -79,7 +79,7 @@ spec:
       hostPID: true
       containers:
         - name: portworx
-          image: portworx/oci-monitor:1.6.1.1
+          image: portworx/oci-monitor:1.6.1.3
```

Comment:
should we also bump the gke examples to ude 1.6.1.3?

Reply:
The new changes only affect vsphere. So not needed. You can still do it for uniformity.